index.js

```diff
@@ -165,7 +165,10 @@ const npmWalker = Class => class Walker extends Class {
 
     if (Array.isArray(pkg.files))
       super.onReadIgnoreFile('package.json', '*\n' + pkg.files.map(
-        f => '!' + f + '\n!' + f.replace(/\/+$/, '') + '/**'
```

Comment:
Would it be a better solution if we change this to always return `'!' + f`, and then update ignore-walker to handle directories when the `/**` is missing? Currently if I do this, the tests for the package-json-bin-x.js tests fail, e.g. https://github.com/npm/npm-packlist/blob/1a4c8bfe712f8b2804c0ae48fb2a9dd99c78ef99/test/package-json-bin-single.js#L24-L31